src/app/_mermaid_common/libs/directives/projectname.dir.js

```diff
@@ -10,7 +10,7 @@ angular.module('mermaid.libs').directive('projectname', [
         watchId: '@'
       },
       template:
-        '<div class="projectname hidden-xs" title="{{project_name_tooltip}}">{{project_name}}</div>',
+        '<div class="projectname" title="{{project_name_tooltip}}">{{project_name}}</div>',
```

Comment:
This seems right, but was there some other reason this div started out hidden?

Reply:
I think the project name used to be right beside the mermaid icon in the top.